src/types/achievements.ts

```diff
@@ -11,6 +12,7 @@ export const achievementDataSchema = z.object({
   botsBusted: z.number(),
   otherPlayers: z.array(playerSchema),
   playerHistory: z.array(matchRoomSchema).optional(),
+  playerAchievements: z.array(userAchievementsSchema).optional(),
```

Comment:
Are you even using this?

Reply:
Yes for every achievement that I call I use this type as import.